blocks/blocks.js

```diff
@@ -359,18 +368,242 @@ class Blocks extends Newsletter {
         this._dragSvg.removeChild(script.elem)
         if (possibleDropTarget && possibleDropTarget.acceptDrop) {
           const { x, y } = script.position
-          possibleDropTarget.acceptDrop(
+          undoEntry.b = possibleDropTarget.acceptDrop(
```

Comment:
Maybe workspaces should get special treatment around here; they currently awkwardly create an entirely new set of methods for dragging, which is not ideal. `acceptDrop` should be used for non-workspaces, such as the sprite equivalent for "Share the love" (or whatever it's called in 3.0), where it's not going to return something for `undoEntry.b`

Reply:
yes